app/controllers/account/appointments_controller.rb

```diff
@@ -0,0 +1,34 @@
+
+module Account
+  class AppointmentController < ApplicationController
+
+    def index
+      @appointments = current_user.appointments
+    end
+
+    def new
+      @appointment = Appointment.new
+    end
+
+    def create
+      @appointment = current_user.appointments.new(appoint_params)
+      if @appointment.save
+        redirect_to root_path
+      else
+        render :new
+      end
+    end
+
+    def destroy
```

Comment:
I think you just copied colled some part of the girlfriends controller without modify it ..... Be more focused next time pls. Time is money